termlink/gsea.py

```diff
@@ -67,16 +68,26 @@ def execute(args):
     Args:
         args:   command line arguments from argparse
     '''
+
     uri = urlparse(args.uri)
     if uri.scheme != 'file':
         raise ValueError(f"uri.scheme '{uri.scheme}' is not supported")
 
-    filename = path.basename(uri.path) 
-    if not match(_filename_regex, filename):
+    input_file = uri.path
+    if not match(_filename_regex, os.path.basename(input_file)):
```

Comment:
It doesn't look like anything has logically changed through this code path. The variable was renamed and the `os.path.basename` call was moved. Is that correct?

Reply:
Yes, I reverted to the former way.  I had gone through a few iterations on abstracting out both the input/output and just didn't clean it up fully once I decided simple is better for now.